docs/dashboard.asciidoc

```diff
@@ -1,2 +1,117 @@
+[float]
 [[dashboard]]
-== Working with Dashboards
\ No newline at end of file
+== Dashboard
+
+A Kibana _dashboard_ displays saved visualizations in groups that you can arrange. You can save a dashboard to share or 
```

Comment:
Went with "displays a set of saved visualizations in groups that you can arrange freely"